mbuild/compound.py

```diff
@@ -2043,31 +2043,34 @@ def energy_minimize(
         tmp_dir = tempfile.mkdtemp()
         original = clone(self)
```

Comment:
May as well delete this line 2044, since it isn't used.

Reply:
openmm energy minimize will use it.  I moved it to so we don't create it if we are doing openbabel energy minimization.